examples/http3-client.rs

```diff
@@ -315,7 +315,19 @@ fn main() {
                 },
             };
 
-            socket.send(&out[..write]).unwrap();
+            match socket.send(&out[..write]) {
```

Comment:
Besides the simplification, you probably want to do the loop+match thing for the initial `socket.send()` call in this file like you did in `client.rs`

Reply:
I shamelessly copied the socket read code which just breaks. Adding a loop-in-loop,I figured, might cause the socket reads (along with other things) to get blocked while trying to send. On windows, send() blocks quite often - at the start of a connection at least 5 iterations...

Reply:
My comment wasn't very clear, but what I meant is that the first `socket.send()` call (the one that starts the connection) should also be changed and use a loop like you did in `client.rs`. That is the one on https://github.com/cloudflare/quiche/blob/master/examples/http3-client.rs#L148 The call linked to this comment should just get the `if let ...` simplification.

Reply:
Got it, fixed.